main.go

```diff
@@ -70,12 +70,15 @@ func mainCmd() error {
 	//
 	// The original ksops plugin is located in the following ways
 	// - Using ${KSOPS_PATH} verbatim.
+	// - Using ${KUSTOMIZE_PLUGIN_HOME}/kustomize/plugin/viaduct.ai/v1/ksops/_ksops.
 	// - Using ${XDG_CONFIG_HOME}/kustomize/plugin/viaduct.ai/v1/ksops/_ksops.
 	// - Using ${HOME}/.config/kustomize/plugin/viaduct.ai/v1/ksops/_ksops.
 	if _, found := os.LookupEnv("KSOPS_DRY_RUN"); !found {
```

Comment:
Extract the placeholder into a var above the main with something like:

```golang
var placeholder = "KSOPS_DRY_RUN_PLACEHOLDER"
```

The modify things here to grab the value:

```suggestion
	if value, found := os.LookupEnv("KSOPS_DRY_RUN"); !found {
```

Then add an else clause on line 93 ([here](https://github.com/joshdk/ksops-dry-run/pull/18/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R93)) to update the placeholder:

```golang
	} else {
		placeholder = value
	}
```